W-OVERFLOW/Chatting
# Fixed Chat Peek warning with SkyHanni

## Description
Currently, the Chatting mod warns the user if they are using SkyHanni's chat peek feature active and have disabled the feature in Chatting. This indicates that this behavior is not intended without giving the user the choice to use either feature.

~~This PR changes the logic to only warn if both mods have the feature enabled to indicate unintended behavior instead of forcing the user to disable SkyHanni's.~~
This PR removes the check entirely.

See related code at https://github.com/hannibal002/SkyHanni/blob/ff6dd291a0f89644508171e6b984295de20466ce/src/main/java/at/hannibal2/skyhanni/features/chat/ChatPeek.kt

## How to test
If you use SkyHanni's chat peek without the PR, leave Chatting's chat peek disabled and see how the notification is displayed.
~~With the PR, you will not see a notification as long as the feature is not enabled in Chatting.~~
With the PR, this message does no longer exist.

## Release Notes

```release-note
```

## Documentation
